apps/iframe/package.json

```diff
@@ -13,14 +13,15 @@
     "start:testnet": "CERAMIC_API=https://ceramic-clay.3boxlabs.com npm run start"
   },
   "dependencies": {
-    "3id-did-provider": "^1.0.2",
     "@3id/common": "^0.1.3",
     "@3id/connect-display": "^0.1.5",
     "@3id/manager": "^0.1.6",
+    "@3id/ui-provider": "file:../../packages/ui-provider",
```

Comment:
That shouldn't be needed if the monorepo is working, have you run `yarn` after adding the package?